app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -14,7 +14,7 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['$http', '$scope',
       { type: "io1", name: "Provisioned IOPS SSD (IO1)" },
       { type: "st1", name: "Throughput Optimized HDD (ST1)" },
       { type: "sc1", name: "Cold HDD (SC1)" },
-      { type: "magnetic", name: "Magnetic" },
+      { type: "standard", name: "Magnetic" },
```

Comment:
Would have really liked if we could fetch this list on-the-fly using some API (AWS API perhaps?)
But don't worry about it, if that's not doable.